app/controllers/concerns/exception_hunter/authorization.rb

```diff
@@ -0,0 +1,23 @@
+module ExceptionHunter
+  module Authorization
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :authenticate_admin_user_class
+    end
+
+    def authenticate_admin_user_class
+      return unless underscored_admin_user_class && !send("current_#{underscored_admin_user_class}")
```

Comment:
Maybe a method to know if authentication is enabled? Somethod simple like 

```ruby
def auth_enabled?
  underscored_admin_user_class.present?
end
```

Could even be added in the config

Reply:
I think the way to know whether authentication is enabled or not was to comment the `:admin_user_class`

```
 # == Dashboard User
  # Exception Hunter allows you to restrict users who can see the dashboard
  # to the ones included in the database. You can change the table name in
  # case you are not satisfied with the default one. You can also remove the
  # configuration if you wish to have no access restrictions for the dashboard.
```

Reply:
should we stick to it or implement an extra config feature?